great_expectations/experimental/datasources/data_asset/data_connector/abs_data_connector.py

```diff
@@ -0,0 +1,123 @@
+from __future__ import annotations
+
+import logging
+import re
+from typing import TYPE_CHECKING, Callable, List, Optional
+
+from great_expectations.core.batch_spec import AzureBatchSpec, PathBatchSpec
+from great_expectations.datasource.data_connector.util import (
+    list_azure_keys,
+    sanitize_prefix,
+)
+from great_expectations.experimental.datasources.data_asset.data_connector import (
+    FilePathDataConnector,
+)
+
+if TYPE_CHECKING:
+    from azure.storage.blob import BlobServiceClient
+
+    from great_expectations.core.batch import BatchDefinition
+
+
+logger = logging.getLogger(__name__)
+
+
+class ABSDataConnector(FilePathDataConnector):
```

Comment:
I'd rather call this `AzureBlobStorageDataConnector`, I think it will be clearer

Reply:
@donaldheppner this is fixed as part #7249 -- thanks.